Author: Gregor Riepl <onitake@gmail.com>
Description: The application metadata path
  has changed a while ago. This patch fixes that.
  Reported upstream: https://github.com/Ultimaker/Cura/issues/2563
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -64,7 +64,7 @@
     install(FILES ${CMAKE_SOURCE_DIR}/resources/images/cura-icon.png
             DESTINATION ${CMAKE_INSTALL_DATADIR}/icons/hicolor/128x128/apps/)
     install(FILES cura.appdata.xml
-            DESTINATION ${CMAKE_INSTALL_DATADIR}/appdata)
+            DESTINATION ${CMAKE_INSTALL_DATADIR}/metainfo)
     install(FILES cura.sharedmimeinfo
             DESTINATION ${CMAKE_INSTALL_DATADIR}/mime/packages/
             RENAME cura.xml )
